### ISSUE TITLE
[Feature] Inform the user regarding expected CSV format

### ISSUE THREAD

**Issue by longnd** [Vivek-sudo/nimble-assessment.backend#8]
## Issue
The application expect the CSV in the following format
```
keyword
{keyword1}
{keyword2}
```
example
```csv
keyword
reactJS
expressJS
hook
```

but there is no clue for the users to know unless reading the code
https://github.com/Vivek-sudo/nimble-assessment.server/blob/713d24ad3c0567152aebc1c5ba9d720fa42d8ac5/src/controllers/keywordController.js#L27-L32

In fact, there are different format the user can upload, including
```
reactjs, expressJS, hook
```

or (without the `keyword` column name)
```
reactJS
expressJS
hook
```
but all of those case will fail without an error message letting the user know.

## Expected
There is a clear instructions for the user regarding the expected CSV format. Ideally, there should be a template file for the user to download.

**Comment by Vivek-sudo** [comment]
Yes this could be done on the frontend side, I had some issues at home and due to which I wasn't able to improve the UX and make it more friendly. But yes this can be and should have been done.

**Comment by Vivek-sudo** [comment]
But I believe this is more of improvments and not implementation issue for the problem statement provided. I focused more on functionality on the problem statement given
